core/emulate_ops.asm

```diff
@@ -237,11 +239,13 @@ fastop_dispatch:
     mov reg_dst, [rsi]
     mov reg_src1, [r10]
     mov reg_src2, [r11]
+    pushf
```

Comment:
Ditto. Move this PUSHF to beginning and adjust stack_arg() offset. Move POPF to end.